packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -42,23 +42,32 @@ export default function CodeTextArea({
   // eslint-disable-next-line
   const fieldProps = otherProps as any;
 
+  const semicolonWarning =
+    "Warning: Please remove any terminating semicolons. GrowthBook uses Common Table Expressions that will break from terminating semicolons.";
```

Comment:
Would love feedback on this error message as I'm not sure I grok the use cases where a semicolon is/is not valid.